README.md

```diff
@@ -33,49 +33,51 @@ Static settings are used on startup and cannot be changed while application is r
-| redis.singleServerConfig.address           | -                  | Redis single server addresses, e.g. "redis://host:port"
-| redis.clusterServersConfig.nodeAddresses   | -                  | Json array with Redis cluster server addresses, e.g. ["redis://host1:port1","redis://host2:port2"]
-| invitations.ttlInSeconds                   | 259200             | Invitation time to live in seconds
-| access.admin.rules                         | -                  | Matches claims from identity providers with the rules to figure out whether a user is allowed to perform admin actions, like deleting any resource or approving a publication. Example: [{"source": "roles", "function": "EQUAL", "targets": ["admin"]}]. If roles contain "admin, the actions are allowed.
+| Setting                                    | Default            |Required| Description
+|--------------------------------------------|--------------------|--------|-----------------------------------------------------------------------------------------------------------
+| config.files                               | aidial.config.json | No|Config files with parts of the whole config.
+| config.reload                              | 60000              | No|Config reload interval in milliseconds.
+| identityProviders                          | -                  | No|Map of identity providers. **Note**. At least one identity provider must be provided.
+| identityProviders.*.jwksUrl                | -                  | Optional|Url to jwks provider. **Required** if `disabledVerifyJwt` is set to `false`. *Note*. Either `jwksUrl` or `userInfoEndpoint` must be provided.
+| identityProviders.*.userInfoEndpoint       | -                  | Optional|Url to user info endpoint. *Note*. Either `jwksUrl` or `userInfoEndpoint` must be provided.  
+| identityProviders.*.rolePath               | -                  | Yes|Path to the claim user roles in JWT token or user info response, e.g. `resource_access.chatbot-ui.roles` or just `roles`.
+| identityProviders.*.loggingKey             | -                  | No|User information to search in claims of JWT token.
+| identityProviders.*.loggingSalt            | -                  | No|Salt to hash user information for logging.
+| identityProviders.*.cacheSize              | 10                 | No|How many JWT tokens to cache.
+| identityProviders.*.cacheExpiration        | 10                 | No| How long to retain JWT token in cache.
+| identityProviders.*.cacheExpirationUnit    | MINUTES            | No|Unit of cache expiration.
+| identityProviders.*.issuerPattern          | -                  | No|Regexp to match the claim "iss" to identity provider
+| identityProviders.*.disableJwtVerification | false              | No|The flag disables JWT verification. *Note*. `userInfoEndpoint` must be unset if the flag is set to `true`.
+| vertx.*                                    | -                  | No|Vertx settings.
```

Comment:
Add example

Reply:
@astsiapanay  need to provide a reference to vertex settings

Reply:
fixed